app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/ui/activities/VendorProfileActivity.java

```diff
@@ -0,0 +1,55 @@
+package com.epfl.neighborfood.neighborfoodandroid.ui.activities;
+
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.epfl.neighborfood.neighborfoodandroid.R;
+
+public class VendorProfileActivity extends AppCompatActivity implements View.OnClickListener{
+
+    ImageView subscribeButton;
+    Boolean subscribed = false;
+    ImageView notificationButton;
+    Boolean notifyOn = false;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_vendor_profile);
+        notificationButton = findViewById(R.id.notificationId);

```

Comment:
As we have viewBindings enabled, we can start using that instead of findViewById (less code to write)

Reply:
I thought about that too however I prefer to keep it this way for this sprint since bindings will be used to set the values as retrieved from the database. I find it really hard working without the database for now.